src/viking.js

```diff
@@ -1,11 +1,106 @@
 // Soldier
-class Soldier {}
+class Soldier {
+  constructor(health, strength) {
+    this.health = health;
+    this.strength = strength;
+  }
+  attack() {
+    return this.strength;
+  }
+  receiveDamage(damage) {
+    this.health -= damage;
+  }
+}
 
 // Viking
-class Viking {}
+class Viking extends Soldier {
+  constructor(name, health, strength) {
+    super(health, strength);
+    this.name = name;
+  }
+  attack() {
+    return this.strength;
+  }
+  receiveDamage(damage) {
+    this.health -= damage;
+    if (this.health <= 0) {
+      return `${this.name} has died in act of combat`;
+    }
+    return `${this.name} has received ${damage} points of damage`;
+  }
+  battleCry() {
+    return "Odin Owns You All!";
+  }
+}
 
 // Saxon
-class Saxon {}
+class Saxon extends Soldier {
+  receiveDamage(damage) {
+    this.health -= damage;
+    if (this.health <= 0) {
+      return "A Saxon has died in combat";
+    }
+    return `A Saxon has received ${damage} points of damage`;
+  }
+}
 
 // War
-class War {}
+class War {
+  constructor() {
+    this.vikingArmy = [];
+    this.saxonArmy = [];
+  }
+  addViking(vikingObj) {
+    this.vikingArmy.push(vikingObj);
+  }
+  addSaxon(saxonObj) {
+    this.saxonArmy.push(saxonObj);
+  }
+  vikingAttack() {
+    let randomViking =
+      this.vikingArmy[Math.floor(Math.random() * this.vikingArmy.length)];
+
+    let randomSaxon =
+      this.saxonArmy[Math.floor(Math.random() * this.saxonArmy.length)];
+
+    let fight1 = randomSaxon.receiveDamage(randomViking.strength);
+    if (randomSaxon.health <= 0) {
+      this.saxonArmy.splice(randomSaxon, 1);
+    }
+    return fight1;
+  }
+
+  /* vikingAttack() {
+    let randomViking = Math.floor(Math.random() * this.vikingArmy.length);
+    let randomSaxon = Math.floor(Math.random() * this.saxonArmy.length);
+    let fight = this.saxonArmy[randomSaxon].receiveDamage(
+      this.vikingArmy[randomViking].strength
+    );
+    if (this.saxonArmy[randomSaxon].health <= 0) {
+      this.saxonArmy.splice([randomSaxon], 1);
+    }
+    return fight;
+  } */
+  saxonAttack() {
+    let randomViking =
+      this.vikingArmy[Math.floor(Math.random() * this.vikingArmy.length)];
+
+    let randomSaxon =
+      this.saxonArmy[Math.floor(Math.random() * this.saxonArmy.length)];
+
+    let fight2 = randomViking.receiveDamage(randomSaxon.strength);
+    if (randomViking.health <= 0) {
+      this.vikingArmy.splice(randomViking, 1);
+    }
+    return fight2;
+  }
+  showStatus() {
+    if (this.saxonArmy.length === 0) {
+      return "Vikings have won the war of the century!";
+    } else if (this.vikingArmy.length === 0) {
+      return "Saxons have fought for their lives and survived another day...";
+    } else if (this.saxonArmy.length === 1 && this.vikingArmy.length === 1) {
```

Comment:
João, here you can use the `else`, since you targeted the two other options with the `if` and the `else if`, you can trigger the `else` operator for the remaining answer.